test/javascripts/src/format24_test.coffee

```diff
@@ -44,3 +44,58 @@ testGroup "format24 (24h time)", ->
       assert.equal getText(el), now.format("h:mma")
       config.useFormat24 = originalFormat24
       done()
+
+  testAsync "turned off relative time", (done) ->
+    ago = moment().subtract("days", 5)
+    day  = ago.format "dddd"
+    time = ago.format "h:mma"
+    expected = "#{day} at #{time}"
+
+    originalFormat24 = config.useFormat24
+    config.useFormat24 = false
+
+    el = addTimeEl type: "time-ago", datetime: ago.toISOString()
+    defer ->
+      assert.equal getText(el), expected
+      config.useFormat24 = originalFormat24
+      done()
```

Comment:
```suggestion
  testAsync "turned off for relative time elements", (done) ->
    ago = moment().subtract("days", 5)
    originalFormat24 = config.useFormat24
    config.useFormat24 = false

    el = addTimeEl type: "time-ago", datetime: ago.toISOString()
    defer ->
      assert.equal getText(el), "#{ago.format("dddd")} at #{ago.format("h:mma")}"
      config.useFormat24 = originalFormat24
      done()
```

This is probably just personal preference. I think the variables here don't communicate any information about the test, they're just acting as data holders with a label. That's fine, but this data isn't complex enough to need a label or anything. So let's just drop them.

Let's make the same edits on the `turned on relative time` test